README.md

```diff
@@ -1,17 +1,25 @@
-# The Algorithms - PHP
+#FOR EDUCATIONAL PURPOSE:
 
-### All algorithms implemented in Php (for education)
+#The Algorithms - PHP
 
-These implementations are for learning purposes. They may be less efficient than the implementations in the Php standard library.
+### All algorithms implemented in Php
 
-## Contribution Guidelines
+The Implementation on this repo are all exaqmples and may in anytime be less efficient than the implementations on the PHP standard Library.
 
-Read our [Contribution Guidelines](CONTRIBUTING.md) before you contribute.
 
-## Community Channel
+## Contribution Guidelines:
+To Contribute to this repo please take your time to read our
+[Contribution Guidelines](CONTRIBUTING.md) before you contribute.
 
-We're on [Gitter](https://gitter.im/TheAlgorithms)!  Please join us.
+## Community Channel
+Let's connect and grow!
+[Gitter](https://gitter.im/TheAlgorithms)!
+[Twitter]
+[Discord]
+Please join us.
```

Comment:
```suggestion
## Community Channels

We're on [Discord](https://discord.gg/c7MnfGFGa6) and [Gitter](https://gitter.im/TheAlgorithms)! Community channels are great for you to ask questions and get help. Please join us!
```

I'd suggest following the [Python](https://github.com/TheAlgorithms/Python/blob/master/README.md#community-channels) (or another `TheAlgorithms` project).